repo2docker/buildpacks/base.py

```diff
@@ -118,6 +118,14 @@
 {% endfor %}
 {% endif -%}
 
+# Run launch script
+{% if launch_scripts -%}
+{% for s in launch_scripts -%}
+RUN chmod +x {{ s }}
+ENTRYPOINT ./{{ s }}
```

Comment:
there can only be one ENTRYPOINT so I think we should error out if there is more than one script before we get here or construct things in a way that doesn't allow us to get into the situation where we have more than one launch script.